babel.config.js

```diff
@@ -11,7 +11,7 @@ module.exports = (api) => {
           modules: false,
           targets: {
             esmodules: true,
-            node: "0.12",
+            node: "14.15.0",
```

Comment:
Keep this, it is for browsers, so let's support all of them

Reply:
```suggestion
            node: "0.12",
```